### PR TITLE
Add basic grunt files

### PR DESCRIPTION
**Pull request by TomaszLemankiewicz**

### CODE REVIEW

File: Projekt dla ambitnych 1/Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+module.exports = function(grunt) {
+
+  // Project configuration.
+  	grunt.initConfig({
+  		sass: {
+     		options: {
```
Comment (matwjablonski) [inline Projekt dla ambitnych 1/Gruntfile.js]: `options:` is a child of `sass:`. so you should add one more tab to it, to show relationship

Comment (matwjablonski) [inline Projekt dla ambitnych 1/Gruntfile.js]: `options:` is a child of `sass:`. so you should add one more tab to it, to show relationship

File: Projekt dla ambitnych 1/Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        	},
+        	dist: {
+        		files: {
+         			 'css/main.css': 'sass/main.sass'
+      	  		}
+    		}
+  	 	},
+
+  	 	imagemin: {
+    		dynamic: {
```
Comment (matwjablonski) [inline Projekt dla ambitnych 1/Gruntfile.js]: `dynamic:` is a child of `imagemin:`. so you should add one more tab to it, to show relationship

File: Projekt dla ambitnych 1/Gruntfile.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+       			 files: ['sass/*.sass'],
+        		 tasks: ['sass'],
+        		 options: {
+            	 	spawn: false,
+        		},
+    		} 
+		},
+
+		browserSync: {
+            dev: {
```
Comment (matwjablonski) [inline Projekt dla ambitnych 1/Gruntfile.js]: clean this part of code

File: Projekt dla ambitnych 1/Gruntfile.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        bsReload: {
+            css: "main.css"
+        }
+    });
+
+  	// Load the plugins tasks 
+  	
+  	grunt.loadNpmTasks('grunt-sass');
+  	grunt.loadNpmTasks('grunt-contrib-imagemin');
+    grunt.loadNpmTasks('grunt-contrib-watch');
```
Comment (matwjablonski) [inline Projekt dla ambitnych 1/Gruntfile.js]: clean this part of code

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: Projekt dla ambitnych 1/Gruntfile.js (hunk 3 above)
Comment (TomaszLemankiewicz) [inline Projekt dla ambitnych 1/Gruntfile.js]: Line 36 or whole browserSync part ?

Comment (TomaszLemankiewicz) [reply Projekt dla ambitnych 1/Gruntfile.js]: Which one ? Line 36 ? Whole browserSync part ?

File: Projekt dla ambitnych 1/Gruntfile.js (hunk 4 above)
Comment (TomaszLemankiewicz) [reply Projekt dla ambitnych 1/Gruntfile.js]: Line 59 ?

Comment (TomaszLemankiewicz) [reply Projekt dla ambitnych 1/Gruntfile.js]: Ok, I see it

File: Projekt dla ambitnych 1/Gruntfile.js (hunk 3 above)
Comment (TomaszLemankiewicz) [reply Projekt dla ambitnych 1/Gruntfile.js]: O I see

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**